### ISSUE TITLE
"Failed to report a downed suspect" Still Bugged (SEF V5)

### ISSUE THREAD

**Issue by sandman332** [eezstreet/SWATEliteForce#146]
Hello,
I am still experiencing this bug as of V5 of SEF.  I get all of my tocs reported, all evidence secured, the mission finishes and I have this penalty.  It still seems to be happening if an officer or another suspect shoots the suspect after he is already restrained and cuffed.  Not sure if something else is now the issue, but this is still happening.

Thanks!

**Comment by theengineertcr** [comment]
This is true. Reporting a sus/civi then killing them or downing them does add a "not reported incap" or death. Can be fixed if theres a check to see if the pawns state changes to incap or death and if they have been reported before, it'll -1 from the current tocs. If they change state and havent been reported, it wont -1 from current tocs. Seems simple enough.

**Comment by eezstreet** [comment]
Seems weird that this would happen. I'll mark it verified though since I just tested it and this seems to be the case.

**Comment by T-MaxWiese-T** [comment]
Tested and can be closed

**Comment by T-MaxWiese-T** [comment]
Tested and verified

**Comment by sandman332** [comment]
Confirmed, still happening.
